### PR TITLE
ci: Fix license header check

### PR DESCRIPTION
**Pull request by iljakuklic**
TODO
* [x] Add missing license headers

### CODE REVIEW

**Comment by TheQuantumPhysicist** [comment]
I'm currently working on this, and doing it right broke things...

**Review by TheQuantumPhysicist — APPROVED** [review]

**Review by muursh — APPROVED** [review]

File: p2p/tests/sync.rs [hunk 1]
```diff
@@ -1,11 +1,11 @@
-// Copyright (c) 2022 RBB S.r.l
+// Copyright (c) 2021 RBB S.r.l
```
Comment (iljakuklic) [inline p2p/tests/sync.rs:1]: Why 2021?

Comment (TheQuantumPhysicist) [reply p2p/tests/sync.rs:1]: Excellent question...

**Review by iljakuklic — COMMENTED** [review]
Looks good. Approving informally. (Cannot approve officially as the original author of the PR.)

**Review by stanislav-tkach — APPROVED** [review]

**Pull request merged**